Example/CryptoOnramp Example/CryptoOnramp Example/CryptoOnramp_ExampleApp.swift

```diff
@@ -24,6 +24,8 @@ struct CryptoOnramp_ExampleApp: App {
                     Color.black.opacity(0.3)
                         .ignoresSafeArea()
                     ProgressView("Loading…")
+                        .padding()
+                        .background(Color(white: 0.9).cornerRadius(8))
```

Comment:
The lengthy request to create the onramp session highlighted how difficult it was to see the loading indicator and text, so I wrapped the `ProgressView` in a more opaque background.

<img width="300" alt="image" src="https://github.com/user-attachments/assets/53a3c32d-14ec-4759-ad60-90c80079db28" />